app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -383,6 +383,27 @@ def handle_successful_status_update(enrollment, response)
     send_verified_email(enrollment.user, enrollment)
   end
 
+  def retro_fail_enrollment(enrollment, response)
+    proofed_at = parse_usps_timestamp(response['transactionEndDateTime'])
+    enrollment_outcomes[:enrollments_failed] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_analytics_attributes(enrollment, complete: true),
+      **response_analytics_attributes(response),
+      passed: false,
+      reason: 'Failed secondary id',
+    )
```

Comment:
It'd be good to mention the value of `reason` here to David and Sumi in case they want to adjust related analytics.

Reply:
changing message based on feedback from David